src/containers/Header.js

```diff
@@ -17,7 +17,46 @@ import HelpButton from '../helpButton';
 
 export function Header(props) {
 	const HeaderService = props.app.locateService("HeaderService");
-	const link = props.brand_image.href ?? "/"
+
+	const createNavbarBrand = () => {
+		const link = props.brand_image.href ?? "/";
+		if (link.includes("http")) {
+			return <AppNavbarBrand
+				href={link}
+				full={{
+					src: props.brand_image.full,
+					alt: props.title,
+					width: 120,
+					height: 30,
+				}}
+				minimized={{
+					src: props.brand_image.minimized,
+					alt: props.title,
+					width: 30,
+					height: 30,
+				}}
+				target="_blank"
+				rel="noopener noreferrer"
+			/>
+		}
+		return <Link to={link}>
+			<AppNavbarBrand
+				tag={'div'}
+				full={{
+					src: props.brand_image.full,
+					alt: props.title,
+					width: 120,
+					height: 30,
+				}}
+				minimized={{
+					src: props.brand_image.minimized,
+					alt: props.title,
+					width: 30,
```

Comment:
In both returns there are almost the same things. Do not repeat yourself. 
In this case you may rewrite this without repeating like this:
Common props in `AppNavbarBrand` you may define in object which you will destructure in both returns.
```
const appNavbarBrandProps = {
  full: {
	  src: props.brand_image.full,
	  alt: props.title,
	  width: 120,
	  height: 30,
  },
  minimized: {
	  src: props.brand_image.minimized,
	  alt: props.title,
	  width: 30,
	  height: 30,
  }
}
...
if (...) <Link...><AppNavbarBrand {{ ...appNavbarBrandProps } ... /></Link>
return <AppNavbarBrand tag={'div'} { ...appNavbarBrandProps } />
```